src/Files.Shared/Utils/IImage.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) 2023 Files Community
+// Licensed under the MIT License. See the LICENSE.
+
+namespace Files.Shared.Utils
+{
+	/// <summary>
+	/// Represents an image which can be displayed in the UI.
```

Comment:
What type of image, is this for thumbnails/icons?

Reply:
Any image. This is a common interface for all images - the implementation holds the necessary information which is all managed by the frontend (UI project). The backend is only exposed to the `IImage` interface and has no internal knowledge of the imaging implementation